src/providers/recentmessages/Impl.cpp

```diff
@@ -105,6 +106,14 @@ QUrl constructRecentMessagesUrl(const QString &name, const int limit)
             QString::number(
                 limit > 0 ? limit : getSettings()->twitchMessageHistoryLimit));
     }
+    if (after > 0 && !urlQuery.hasQueryItem("after"))
```

Comment:
consider `std::optional` here? Magic value of zero to denote disabling should at least be in the comment above the method otherwise

Reply:
Maybe a datetime object/"instant" datatype could work better here as well. Otherwise you should probably specify in the comment that these are millisecond-since-epoch timestamps. Consider `uint64_t` as well, since `long` is 32-bit on many 32-bit systems (not sure if Chatterino supports these targets though)

Reply:
Switched to int64_t and updated the param documentation :+1: